units/event_metajob.go

```diff
@@ -339,3 +190,8 @@ func (j *eventMetaJob) logEventCount() error {
 
 	return nil
 }
+
+func sha256sum(ids []string) string {
```

Comment:
sort the strings so you get a stable hash

Reply:
done